emctl/cmd/client/command/flags/flags.go

```diff
@@ -251,6 +263,10 @@ func (i *Install) AttachCmd(cmd *cobra.Command) {
 
 	cmd.Flags().IntVar(&i.EasegressControlPlaneReplicas, "easemesh-control-plane-replicas", DefaultMeshControlPlaneReplicas, "Mesh control plane replicas")
 	cmd.Flags().IntVar(&i.MeshIngressReplicas, "easemesh-ingress-replicas", DefaultMeshIngressReplicas, "Mesh ingress controller replicas")
+	cmd.Flags().BoolVar(&i.OnlyAddOn, "only-add-on", false, "Only install add-ons")
+	cmd.Flags().StringArrayVar(&i.AddOns, "add-ons", []string{}, "Names of add-ons to be installed")
+	cmd.Flags().StringVar(&i.ShadowServiceControllerImage, "shadowservice-controller-image", DefaultShadowServiceControllerImage, "Shadow service controller image name")
+	cmd.Flags().IntVar(&i.ShadowServiceControllerReplicas, "shadowservice-controller-replicas", DefaultShadowServiceControllerReplicas, "Shadow service controller replicas")
```

Comment:
Will multiple replicas will cause distributed race conditions?

Reply:
Talked with @zouyingjie , it should be ok for the shadow service controller to have replicas > 1, but it is not very useful, so I will hard code the value to 1.